lib/osf-components/addon/components/ancestry-display/template.hbs

```diff
@@ -0,0 +1,17 @@
+<span local-class='Titles'>
+    {{#each this.ancestry as |ancestor index|}}
+        <span local-class='AncestorTitle'>
+            {{#if (and this.useLinks ancestor.id)}}
+                <OsfLink
+                    data-test-ancestor-title={{index}}
+                    @href='/{{ancestor.id}}'
```

Comment:
Try to save `@href` for non-osf links. When linking to a guid like this, you can transition to the `resolve-guid` route. It'll figure out the right thing to do.
```hbs
<OsfLink
  @route='resolve-guid'
  @models={{array ancestor.id}}
>
```